harvard-lts/CURIOSity
# Adding focus state border to btn-reset.

**Adding focus state border to btn-reset.**
* * *

**JIRA Ticket**: [LTSMAINT-682](https://jira.huit.harvard.edu/browse/LTSMAINT-682)

# What does this Pull Request do?
Adds a focus state to the clear search icon on the browse page.

# How should this be tested?

A description of what steps someone could take to:
* Rebuild container using `search-bar-focus` branch
* Go to: https://localhost:21407/demo/browse/ceramics
* Type something into the search box
* Tab to the clear search icon and confirm that a border appears on focus.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
